components/button/button-add.js

```diff
@@ -155,7 +155,7 @@ class ButtonAdd extends RtlMixin(PropertyRequiredMixin(FocusMixin(LocalizeCoreEl
 	constructor() {
 		super();
 
-		this.mode = MODE.ICON;
+		this.mode = MODE.icon;
```

Comment:
I lowercased these because in Daylight it was using an allcaps version for the default value